Marlin/src/feature/easythreed_ui.cpp

```diff
@@ -142,7 +142,7 @@ void EasythreedUI::loadButton() {
       }
       else if (!flag) {
         flag = true;
-        queue.inject(!READ(BTN_RETRACT) ? F("G91\nG0 E10 F180\nG0 E-120 F180\nM104 S0") : F("G91\nG0 E100 F120\nM104 S0"));
+        queue.inject(!READ(BTN_RETRACT) ? F("G91\nG0 E10 F180\nG0 E-220 F180\nM104 S0") : F("G91\nG0 E200 F120\nM104 S0"));
```

Comment:
Is it possible to use this UI with other machines besides those which prefer these lengths?